src/main/java/com/example/flipcardsapp/FlipCard/FlipCardFront.java

```diff
@@ -0,0 +1,33 @@
+package com.example.flipcardsapp.FlipCard;
+
+import javax.persistence.*;
+
+@Entity
+public class FlipCardFront implements FlipCard{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    private String content;
+    private StudyLevel level;
+
+    public FlipCardFront(String content) {
+        this.content = content;
+        this.level = StudyLevel.BAD;
```

Comment:
Is it always a constant value?
maybe better idea is assigning this in method field